libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/infopanel/InfoPanelHeaderRoutesPreviewBinder.kt

```diff
@@ -0,0 +1,37 @@
+package com.mapbox.navigation.dropin.binder.infopanel
+
+import android.transition.Scene
+import android.transition.TransitionManager
+import android.view.ViewGroup
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.internal.extensions.navigationListOf
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+import com.mapbox.navigation.dropin.NavigationViewContext
+import com.mapbox.navigation.dropin.R
+import com.mapbox.navigation.dropin.databinding.MapboxInfoPanelHeaderRoutePreviewLayoutBinding
+import com.mapbox.navigation.dropin.internal.poiNameComponent
+import com.mapbox.navigation.dropin.internal.startNavigationButtonComponent
+import com.mapbox.navigation.ui.base.lifecycle.UIBinder
+
+@ExperimentalPreviewMapboxNavigationAPI
+internal class InfoPanelHeaderRoutesPreviewBinder(
+    private val context: NavigationViewContext
+) : UIBinder {
+
+    override fun bind(viewGroup: ViewGroup): MapboxNavigationObserver {
+        val scene = Scene.getSceneForLayout(
+            viewGroup,
+            R.layout.mapbox_info_panel_header_route_preview_layout,
+            viewGroup.context
+        )
+        TransitionManager.go(scene)
+        val binding = MapboxInfoPanelHeaderRoutePreviewLayoutBinding.bind(viewGroup)
+
+        return context.run {
+            navigationListOf(
+                poiNameComponent(binding.poiName),
```

Comment:
In `RoutePreview` state we shouldn't show POI details. Instead we should show trip progress information along with start navigation button.

| On `main` branch(Correct) | On `this` branch(Incorrect) |
| ------------------ | ----------------- |
| <img src="https://user-images.githubusercontent.com/9770186/190060490-39571b1b-1d97-43b5-a193-2cf7a143d472.jpg" width="200"/> | <img src="https://user-images.githubusercontent.com/9770186/190060482-6186c8e4-d4cc-435a-985c-3b261540de74.jpg" width="200"/> |

@tomaszrybakiewicz

Reply:
Good catch. I missed that when splitting the logic.